data/json/monster_attacks.json

```diff
@@ -38,7 +40,9 @@
     "body_parts": [ [ "FOOT_L", 2 ],[ "FOOT_R", 2 ], [ "LEG_L", 3 ], [ "LEG_R", 3 ], [ "HAND_L", 2 ], [ "HAND_R", 2 ], [ "HEAD", 3 ], [ "EYES", 2 ], [ "MOUTH", 1 ], [ "ARM_L", 3 ], [ "ARM_R", 3 ], [ "TORSO", 4 ] ],
     "hit_dmg_u": "The %1$s claws at you!",
     "hit_dmg_npc": "The %1$s claws at <npcname>!",
+    "hit_dmg_monster": "The %1$s claws the <npcname>!",
```

Comment:
Doesn't need "the"

Reply:
It is 'The zombie claws the fish' vs 'the zombie claws fish', isn't the first better?

Reply:
This is the current behaviour: http://prntscr.com/ikhbtw

Reply:
Aha, `<npcname>` threw me off. Got it.

Reply:
Hmm while testing it further it seems like <npcname> adds 'the' when it isn't there already.
("A blow from the %s sends <npcname> flying!")  became "A blow from the hulk sends the debug monster flying!"